### DISCUSSION TITLE
When doing list-settings with pac org the error message is not intuitive

### DISCUSSION THREAD

**Discussion by kkanakas** [microsoft/powerplatform-build-tools#451] [General]
I am doing a pac list-settings, I had assumed that I could use my existing auth profile, since I had not done an pac org select prior and just ran pac org list-settings. I got the following unintuitive error message

Sequence of steps : 
1) Luanch PowerShell 
2) type Pac org list-settings
obscure error message 
 -----------------------
 kartikka@neelix ~ pac org list-settings --filter IP
Sorry, the app encountered a non recoverable error and will need to terminate. The exception details have been captured and will be forwarded to the development team, if telemetry has been enabled. Exception Id: c75b8eea-bc6d-4176-a335-6191252f6d65, Exception Type: System.ArgumentNullException
The diagnostic logs can be found at: C:\Users\kartikka\.dotnet\tools\.store\microsoft.powerapps.cli.tool\1.25.5\microsoft.powerapps.cli.tool\1.25.5\tools\net6.0\any\logs\pac-log.txt

<img width="901" alt="image" src="https://github.com/microsoft/powerplatform-build-tools/assets/22942069/488593cb-6aae-481e-b119-e81858739865">

once I do a pac org select,then everything works. 
---------------------------------------

can this please be fixed

**Comment by petrochuk** [comment]
Thanks for reporting @kkanakas. It'll be fixed in the next version
